content/en/community/end-user/_index.md

```diff
@@ -1,20 +1,24 @@
 ---
 title: End-user Resources
 linkTitle: End User
-description: Vendor-agnostic resources for users of OpenTelemetry
+description: Vendor-agnostic resources for OpenTelemetry users
 ---
 
-The [OpenTelemetry End User Working Group][] has heard feedback from users who
-desire a vendor-agnostic space to discuss adopting OpenTelemetry, so we provide
-multiple ways for you to connect with other users of the project and share best
-practices:
+The
+[OpenTelemetry End User SIG](https://github.com/open-telemetry/community/tree/main/working-groups/end-user)
+has heard feedback from users who desire a vendor-agnostic space to discuss
+OpenTelemetry adoption and implementation, so we provide the following ways for
+you to connect with other users of the project, learn about OTel, and provide
+feedback about your OTel usage:
```

Comment:
Sounds a bit like summoning a deity to me. :)

How about "If you're looking for a vendor-agnostic space to..."

I don't think users are concerned with how this came to happen, but at the end you could link the working group as the source of this initiative, or for learning more.

Reply:
You're right, I felt it was a bit weird too but was going with what was already written... I rewrote it! Hopefully it reads better now, lmk what you think!